releases/release-1.26/release-notes/maps/pr-110268-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 110268
+releasenote:
+  text: |-
+    'The iptables kube-proxy backend now process service/endpoint changes
```

Comment:
I think process => processes makes more sense

Reply:
I think it is in the second person since it is referring to iptables (in plural)